### PR TITLE
Added an example MagicDraw command line plugin + created common build

### PR DESCRIPTION
**Pull request by dandrid**

### CODE REVIEW

File: com.incquerylabs.magicdraw.plugin.example/build.gradle [hunk 1]
```diff
@@ -1,252 +1,9 @@
...
-
-def os = org.gradle.internal.os.OperatingSystem.current()
-def platformConfigSetting = 
-        (os.isWindows())
-		? ['-DWINCONFIG=true']
-		: ((os.isMacOsX())
-			? ['-DWINCONFIG=false', '-Xdock:name=MagicDraw', '-Xdock:icon=bin/md.icns', '-Dapple.laf.useScreenMenuBar=true']
-: ['-DWINCONFIG=false'])
-
 tasks.withType(JavaExec) {
```
Comment (thSoft) [inline com.incquerylabs.magicdraw.plugin.example/build.gradle]: This part is common in both plugins, please extract it to the root build script.

**Review by thSoft — COMMENTED** [review]
This is a useful enhancement, however please address my feedback.

**Comment by thSoft** [comment]
LGTM. @lunkpeter, please also approve.

**Comment by lunkpeter** [comment]
We have added CLA and test execution support in #20 we can close this PR

**Pull request closed without merging**